README.md

```diff
@@ -1,5 +1,5 @@
-[![Build Status](https://travis-ci.org/OCA/sale-reporting.svg?branch=8.0)](https://travis-ci.org/OCA/sale-reporting)
-[![Coverage Status](https://coveralls.io/repos/OCA/sale-reporting/badge.png?branch=8.0)](https://coveralls.io/r/OCA/sale-reporting?branch=8.0)
+[![Build Status](https://travis-ci.org/OCA/sale-reporting.svg?branch=9.0)](https://travis-ci.org/OCA/sale-reporting)
+[![Coverage Status](https://coveralls.io/repos/OCA/sale-reporting/badge.png?branch=9.0)](https://coveralls.io/r/OCA/sale-reporting?branch=9.0)
```

Comment:
Strange this is not done in the last commit, not sure when and why it was changed.